src/ResumeCache.h

```diff
@@ -93,7 +96,8 @@ class ResumeCache {
   // Inferred position of the rcvd frames
   ResumePosition impliedPosition_{0};
 
-  std::unordered_map<StreamId, ResumePosition> streamMap_;
+  // Only active REQUEST_STREAMs are preserved here
+  std::set<StreamId> activeStreams_;
```

Comment:
Why does a sorted set make testing easier? If you need it sorted you can just do that in the test?